web/src/components/query-options-dropdown.tsx

```diff
@@ -30,6 +32,17 @@ export const QueryOptionsPanel: React.FC<QueryOptionsDropdownProps> = ({ options
     }
   ];
 
+  const matchOptions: MatchOption[] = [
+    {
+      label: t('Match all'),
+      value: 'all'
+    },
+    {
+      label: t('Match any'),
+      value: 'any'
```

Comment:
Your translations are missing from `/locales/en/plugin__network-observability-plugin.json`

I guess we should auto generate them when we build if everybody agrees

Reply:
ah yeah! I forgot to run the i18n task